### DISCUSSION TITLE
development setup error [Jul 29]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3026] [Discord Threads]
> 🙋‍♂️ **findzhang**  
## who can help me?  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1001786935779524719/unknown.png)

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### what version are you trying? also ensure port 8080 isn't occupied  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1001786936710664264)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
